app/src/test/java/com/weatherxm/util/ChartsAxisFormattersTest.kt

```diff
@@ -24,12 +41,12 @@ class ChartsAxisFormattersTest : BehaviorSpec({
             When("it is less than 10000") {
                 and("The formatter does not support decimals") {
                     then("it should return the correct value") {
-                        yAxisFormatter.getAxisLabel(1000.65F, null) shouldBe "1001"
+                        yAxisFormatter.getAxisLabel(1000.65F, null) shouldBe "1,001"
                     }
                 }
                 and("The formatter supports decimals") {
                     then("it should return the correct value") {
-                        yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1000.7"
+                        yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1,000.7"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/WeatherXM/wxm-android/pull/342#issuecomment-2388448877>#04ac52</a><span> - 10/02/2024, 12:12 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The test case assumes a specific locale (likely US) for the thousands separator, which may not be consistent across all locales.
</div>
<div id="fix">
<b>Fix:</b> Update the test case to use a specific locale for testing, and add additional test cases for different locales to ensure consistent behavior across various number formatting conventions.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -46,7 +46,15 @@
              }
              and("The formatter supports decimals") {
                  then("it should return the correct value") {
 -                    yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1,000.7"
 +                    withLocale(Locale.US) {
 +                        yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1,000.7"
 +                    }
 +                    withLocale(Locale.GERMANY) {
 +                        yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1.000,7"
 +                    }
 +                    withLocale(Locale.FRANCE) {
 +                        yAxisFormatterWithDecimals.getAxisLabel(1000.65F, null) shouldBe "1 000,7"
 +                    }
                  }
              }
          }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged